GameLobby.php

```diff
@@ -78,7 +78,7 @@
   <meta http-equiv="content-type" content="text/html; charset=utf-8" >
   <title>Game Lobby</title>
   <link id="icon" rel="shortcut icon" type="image/png" href="./Images/<?= $icon ?>"/>
-  <link rel="stylesheet" href="./css/karabast122724.css">
+  <link rel="stylesheet" href="./css/karabast122725.css">
```

Comment:
the naming convention is MMDDYY
you can just do the next day, 122824

Reply:
Got it, I've fixed it at #470